src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -123,12 +123,19 @@ parameters:
     description: The identifier of a target node.
     required: true
     type: string
-  nodeFolderIdParam:
-    name: nodeId
-    in: path
-    description: The identifier of a folder node.
-    required: true
-    type: string
+  maxItemsForSizeParam:
+    name: maxItems
+    in: query
+    description: |
+      This assists in handling scenarios where ***large files*** or ***subfolders*** exist within a folder,
+      allowing the folder size determination mechanism to function in chunks.
+      
+      So that calculating the huge folder will be done efficiently.  
+      If not supplied then the default value is 100.
```

Comment:
Is there a server-side maximum too? Otherwise I think this provides a DoS method for authenticated users.

Reply:
We will set a maximum value of **1000** on the server side for partitioning.
We started working on it.